src/components/tx/DecodedTx/index.tsx

```diff
@@ -78,13 +78,17 @@ const DecodedTx = ({
       )}
 
       <Accordion elevation={0} onChange={onChangeExpand} sx={!tx ? { pointerEvents: 'none' } : undefined}>
-        <AccordionSummary expandIcon={<ExpandMoreIcon />} className={accordionCss.accordion}>
+        <AccordionSummary
+          data-testid="accordion-summary"
```

Comment:
I would name it something more specific to decoded txs as we use the same AccordionSummary in many other components.

Reply:
Done